Formula/d/davmail.rb

```diff
@@ -5,6 +5,14 @@ class Davmail < Formula
   sha256 "fc9b0916d19a94db8ecf6d5e7823d1fcbbcbb9e32d40047b55228df12a673ef0"
   license "GPL-2.0-or-later"
 
+  livecheck do
+    url "https://sourceforge.net/projects/davmail/rss?path=/davmail"
+    regex(%r{url=.*?/DavMail-MacOSX[._-]v?(\d+(?:\.\d+)+)-(\d+(?:\.\d+)*)\.app\.zip}i)
+    strategy :sourceforge do |page, regex|
+      page.scan(regex).map { |match| "#{match[0]}-#{match[1]}" }
+    end
```

Comment:
```suggestion
    regex(%r{url=.*?/davmail[._-]v?(\d+(?:[.-]\d+)+)\.(?:t|zip)}i)
```

We're not using `DavMail-MacOSX-6.1.0-3423.app.zip` in the formula. If that's intentional, then we should be checking for the zip files like `davmail-6.1.0-3423.zip` instead.

For what it's worth, modifying the standard regex to use `[.-]` in the repeating part like `v?(\d+(?:[.-]\d+)+)` will handle versions like `6.1.0-3423` without needing a `strategy` block. There are some rare cases where we need something more explicit (i.e., the looseness matches more than we want) but it's fine here.

Reply:
Clever, thanks for the tip!